samples/sampler/stories/qa/Text.js

```diff
@@ -91,6 +146,79 @@ export const TallGlyphSupportInComponents = () => {
 					<br />
 					<Header alt="Header Compact" type="compact" title={children} subtitle={children} />
 				</Section>
+
+				<Section title="Different components with tall characters as children">
+					<Button
+						alt="Button"
+						onClick={action('onClick')}
+						backgroundOpacity={select('backgroundOpacity', prop.backgroundOpacity, Button)}
+						disabled={boolean('disabled', Button)}
+						focusEffect={select('focusEffect', prop.focusEffect, Button)}
+						icon={select('icon', prop.icons, Button)}
+						minWidth={boolean('minWidth', Button, true) ? void 0 : false}
+						selected={boolean('selected', Button)}
+						size={select('size', ['small', 'large'], Button)}
+					>
```

Comment:
Button is already in <Section title="Basic Form controls" size="50%">
Please don't make any duplication.

Reply:
I suddenly wanted to think more about the direction, please do not proceed for a while

Reply:
Please remove duplicated component and migrate the Knob to the existing one.
In other words, Please remove this part and add knob to line 122.

Reply:
Okay. I also moved the knobs to checkboxItem from above.